File: app/src/main/java/com/so/luotk/models/youtubeEx/youtube/playerResponse/RunsItem.java
package com.so.luotk.models.youtubeEx.youtube.playerResponse;

import java.io.Serializable;

public class RunsItem implements Serializable {
    private String text;

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    @Override
    public String toString() {
        return
                "RunsItem{" +
                        "text = '" + text + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/so/luotk/models/youtubeEx/newModels/RunsItem.kt
package com.so.luotk.models.youtubeEx.newModels

import com.google.gson.annotations.SerializedName

class RunsItem {
    @SerializedName("text")
    var text: String? = null
    override fun toString(): String {
        return "RunsItem{" +
                "text = '" + text + '\'' +
                "}"
    }
}